src/app/(app)/_components/carousel/index.tsx

```diff
@@ -0,0 +1,51 @@
+"use client";
+
+/* eslint-disable react/no-array-index-key */
+import { animate, useMotionValue, motion } from "framer-motion";
+import { useEffect } from "react";
+import useMeasure from "react-use-measure";
+import Card from "@/src/app/(app)/_components/card";
+import { images } from "@/src/lib/data";
+
+export default function Page({ flag }: { flag: boolean }) {
```

Comment:
```suggestion
export default function Page({ flag = true }: { flag?: boolean }) {
```

のようにすると、trueの場合はpropsを省略できます。（知っていて意図的に渡していれば大丈夫です！）

```tsx
<Carousel  />
```
👆のように渡さなくても`true`になる